index.js

```diff
@@ -0,0 +1,15 @@
+const electron = require('electron')
```

Comment:
please keep this PR focused on one subject, move electron to a separate PR

Reply:
oops didn't mean to add it, Don't know how it ended up there. Will remove it and fix remaining reviews.